src/java.base/share/classes/java/lang/ThreadLocal.java

```diff
@@ -807,4 +798,43 @@ private void expungeStaleEntries() {
             }
         }
     }
+
+
+    /**
+     * Reads the value of the jdk.traceVirtualThreadLocals property to determine if
+     * a stack trace should be printed when a virtual threads sets a thread local.
```

Comment:
Trivial typo - "virtual threads sets ...." should have been "virtual thread sets ...."